src/components/text-editor/text-editor.style.js

```diff
@@ -14,6 +14,7 @@ StyledEditorWrapper.defaultProps = {
 };
 
 const StyledEditorContainer = styled.div`
+  border-radius: var(--borderRadius050);
```

Comment:
You'll need to add the same radius to `StyledEditorOutline` or the focus state will not have rounded corners

![image](https://user-images.githubusercontent.com/44157880/218138165-05376562-b98a-454a-a936-b7fc174972ca.png)

vs

![image](https://user-images.githubusercontent.com/44157880/218137992-d294eb26-3ae7-49ac-b8ea-6f61873edcf0.png)